src/main/webapp/assets/modules/common/vnwConfigService.js

```diff
@@ -667,7 +706,12 @@ techlooper.factory("vnwConfigService", function (jsonValue, $translate, $rootSco
       $.each(locations, function (i, location) {if (location.location_id == locationId) {return (text = location[vnwLang]);}});
       return text;
     },
-
+    getCompanySizeText: function (companySizeId) {
+      if (!companySizeId) return '';
+      var text = "";
+      $.each(companySizes, function (i, item) {if (item.id == companySizeId) {return text = item.value;}});
```

Comment:
Line is too long.<br>Did you mean to return a conditional instead of an assignment?